assopy/models.py

```diff
@@ -50,23 +51,8 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
-def _gravatar(email, size=80, default='identicon', rating='r'):
-    # import code for encoding urls and generating md5 hashes
-    import urllib.request, urllib.parse, urllib.error, hashlib
-
-    lowercase_email = email.lower()
-    if not isinstance(lowercase_email, bytes):
-        # Encode it!
-        lowercase_email = lowercase_email.encode('utf-8')
-
-    gravatar_url = "http://www.gravatar.com/avatar/" + hashlib.md5(lowercase_email).hexdigest() + "?"
-    gravatar_url += urllib.parse.urlencode({
-        'default': default,
-        'size': size,
-        'rating': rating,
-    })
-
-    return gravatar_url
+# Do we need this? It is here just for compatibility reasons
+_gravatar = gravatar
```

Comment:
This doesn't seem to be used anywhere in this file or imported from this file, it'd be a good opportunity to remove it :)

Reply:
Perfect! Removed.